### ISSUE TITLE
Suggestion: Emergency Sails

### ISSUE THREAD

**Issue by CurePindal** [thindil/steamsky#50]
Emergency sails

So, when a ship is stranded e.g. with a dead engine and you cant repair it - why should that be the end of the game? 
Why shouldnt the crew be able to set up some Robinson's raft-like emergency sails from rags/cloth/etc to limp back to a port?
Its still a struggle of time vs. food/water and an better option that a cold dead end of the game.

**Comment by thindil** [comment]
Hello, and thank you for your proposition :) I just have some question, how it could looks:
1. It should be as a ship module, an item bought in bases or just appear from nowhere (exactly this rags)?
2. It should have an infinite (limited by food/drinks) range or limited?
And one thing - I can't promise that this feature will land soon in the game, as you probably saw, my TODO list for the game is very long :)

**Comment by Asmageddon** [comment]
What I'd really like is the ability to cannibalize items and modules for materials, and to use them in various ways - repair some of the modules, create makeshift versions of others, create an emergency radio or whatnot else.

**Comment by thindil** [comment]
It is probably bit out of context :) But slowly it appearing in the game :) Deconstructing just arrived, other "makeshifts" things probably will appear also later.

**Comment by ChrisEdBurt** [comment]
Like Asmageddon suggested, the ability to cannibalize certain modules to make a temporary sail is a good idea, particularly the cabins as I imagine it would contain furniture with some kind of fabric or cloth. 
Another option could be by damaged your cockpit you can piece together a barely functional radio that you can use to call for help, maybe someone will come to your rescue, maybe they try to board your ship and kill you, who knows?
It works as a game mechanic but also from a logical perspective as well.
You could reduce the quality of the cabin and use it to make a sail, you'd then have to upgrade it again to get the original quality back.

**Comment by thindil** [comment]
Yes, I will think about it, especially that I have on Wish List something similar. Maybe not exactly "makeshifts" things, but from emergency vehicle to simple option to disable it in the game configuration ;)

**Comment by rmvgaines** [comment]
You might look at how Star Control II handled this. Basically, your ship sits dead in the sky and various "events" happen: attacks by pirates, other ships offering to sell fuel at high prices, and (rarely) free fuel from passing ships. In Star Control II, it added a lot of tension to the game. There could also be a faction reputation penalty associated with running out of fuel. I do like the idea of being able to purchase emergency sails, but this could sit alongside of them before the sails are purchased or if they get damaged.

**Comment by thindil** [comment]
Thank you for the ideas, I will think about them. I have some thoughts I wish to test here. The main reason why I'm still postponing the work on the issue is, I want to do it as a module, so players will be able to enable or disable it, without polluting the game configuration. But this requires to do some kind of UI for modules too. And I'm constantly distracted from this issue by others. 😉

**Comment by rmvgaines** [comment]
Modularity is always good.
